src/ensembl/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable rules that conflict with prettier
+    'plugin:prettier/recommended' // Displays prettier errors as ESLint errors. This has to be the last in this array (not yet sure why).
```

Comment:
Do we even want to display `prettier` errors? Shouldn't prettier just format the code without making us even think about it?

_(Personally, I am not a fan of `prettier`, because I often find its formatted code harder to read than the original, but if we must use `prettier`, then perhaps it should just do its work quietly?)_

Reply:
We can get rid of prettier if we think it is too intrusive. However, I think it is good to know that the format is a bit off in my code, though, it does format the code for us when we commit.

Reply:
Turns out this actually helps to silence all those formatting errors that ESLint complains about 😁 I changed the value of `prettier/prettier` (all prettier rules) from `1` to `0` to off it and voila, no more formatting errors.